Gemfile.lock

```diff
@@ -436,14 +427,14 @@ GEM
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    webpacker (4.0.7)
+    webpacker (4.2.2)
```

Comment:
### [webpacker](https://github.com/rails/webpacker)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.0.7 → 4.2.2

[_[change-log](https://github.com/rails/webpacker/blob/v4.2.2/CHANGELOG.md), [source-code](https://github.com/rails/webpacker/tree/v4.2.2)_]





<details>
<summary>Commits</summary>

A change of **89** commits. See the full changes on [the compare page](https://github.com/rails/webpacker/compare/v4.0.7...v4.2.2).

These are the first 10 commits:
- (195396f) [Record the compilation digest even on webpack error (#2117)](https://github.com/rails/webpacker/commit/195396f1e6dd1cb94d4c12887ae40884a6c18505)
- (6dfcad6) [Update webpacker config to not include require('babel-plugin…](https://github.com/rails/webpacker/commit/6dfcad63977585beffd41c6724c8c9b48c1a9b69)
- (f0dcce2) [Adds troubleshooting hint to docs (#2123)](https://github.com/rails/webpacker/commit/f0dcce2233cc470aa535fff64ec9100acf0fef66)
- (4787903) [Move bundle arguments to bundle install command (#2108)](https://github.com/rails/webpacker/commit/478790319d23d98dd75a441c1823336be54dddfb)
- (53129ef) [Adding Lazy Loading documentation with examples (#2118)](https://github.com/rails/webpacker/commit/53129ef636d5d5f82479555d46d3c2242596efc2)
- (818b973) [reduce indirection to docs](https://github.com/rails/webpacker/commit/818b9737f43fbf989b8794cd78c5d98fc62f3de3)
- (acadd1e) [include changes suggested by @jakeNiemiec](https://github.com/rails/webpacker/commit/acadd1ec88a58664748c40dda1fdf4649c5a8eff)
- (cfd89e8) [include further changes suggested by @jakeNiemiec](https://github.com/rails/webpacker/commit/cfd89e80015f1285c4359161a098a1939d17f616)
- (981753a) [Merge pull request #2128 from lazylester/patch-2](https://github.com/rails/webpacker/commit/981753a87c9060c8ff99716123cdcba2950d18e0)
- (66c3b44) [Svelte](https://github.com/rails/webpacker/commit/66c3b449725d802d83f0f0ff3d9d0539394d05ad)


</details>